intermediate/data_cleaning/05.1_intro.md

```diff
@@ -0,0 +1,79 @@
+# Data Tidying
+
+Array data that are represented by Xarray objects are often multivariate, multi-dimensional, and very complex. Part of the beauty of Xarray is that it is adaptable and scalable to represent a large number of data structures. However, this can also introduce difficulty (especially for learning users) in arriving at a workable structure that will best suit one's analytical needs.
```

Comment:
I would add a 
```
:::{seealso}
Looks for examples [here](ADD LINK to EXAMPLES.MD)
:::
```

you might need to replace `:::` with ```

Reply:
on it !